main.go

```diff
@@ -98,11 +98,30 @@ func avsSyncMain(cliCtx *cli.Context) error {
 	for _, quorum := range cliCtx.IntSlice(QuorumListFlag.Name) {
 		quorums = append(quorums, byte(quorum))
 	}
+
+	firstSyncTimeStr := cliCtx.String(FirstSyncTimeFlag.Name)
+	var sleepBeforeFirstSyncDuration time.Duration
+	if firstSyncTimeStr == "" {
+		sleepBeforeFirstSyncDuration = 0 * time.Second
+	} else {
+		now := time.Now()
+		firstSyncTime, err := time.Parse("15:04:05", firstSyncTimeStr)
```

Comment:
nit: use `time.TimeOnly`

also nit: would taking a unix timestamp easier and less brittle to handle the timing?